internal/runnerinstall/nomad.go

```diff
@@ -73,13 +74,16 @@ func (i *NomadRunnerInstaller) Install(ctx context.Context, opts *InstallOpts) e
 		if i.Config.HostVolume != "" {
 			return fmt.Errorf("choose either CSI or host volume, not both")
 		}
+		if i.Config.CsiVolumeName == "" {
+			i.Config.CsiVolumeName = "waypoint-" + opts.Id + "-runner"
```

Comment:
I think we could use the new function `defaultRunnerName` here, in `runnerinstall.go`.

Reply:
Resolved in `68a0515c2`